interface/.dockerignore

```diff
@@ -1,4 +1,3 @@
 bin
 build
-node_modules
-vendor
+adage/adage/config.py
```

Comment:
This one isn't needed, right? It's the one in the root-level `.dockerignore` that we want I think.